src/cli/backup/backup.go

```diff
@@ -202,48 +203,53 @@ func runBackups(
 	selectorSet []selectors.Selector,
 ) error {
 	var (
-		merrs *multierror.Error
-		bIDs  []model.StableID
+		bIDs []model.StableID
+		errs = []error{}
 	)
 
 	for _, discSel := range selectorSet {
-		bo, err := r.NewBackup(ctx, discSel)
+		var (
+			owner = discSel.DiscreteOwner
+			bctx  = clues.Add(ctx, "resource_owner", owner)
+		)
+
+		bo, err := r.NewBackup(bctx, discSel)
 		if err != nil {
-			merrs = multierror.Append(merrs, errors.Wrapf(
-				err,
-				"Failed to initialize %s backup for %s %s",
-				serviceName,
-				resourceOwnerType,
-				discSel.DiscreteOwner))
+			errs = append(errs, clues.Wrap(err, owner).WithClues(bctx))
+			Errf(bctx, "%v\n", err)
 
 			continue
 		}
 
-		err = bo.Run(ctx)
+		err = bo.Run(bctx)
 		if err != nil {
-			merrs = multierror.Append(merrs, errors.Wrapf(
-				err,
-				"Failed to run %s backup for %s %s",
-				serviceName,
-				resourceOwnerType,
-				discSel.DiscreteOwner))
+			errs = append(errs, clues.Wrap(err, owner).WithClues(bctx))
+			Errf(bctx, "%v\n", err)
 
 			continue
 		}
 
 		bIDs = append(bIDs, bo.Results.BackupID)
+		Infof(ctx, "Done - ID: %v\n", bo.Results.BackupID)
 	}
 
-	bups, ferrs := r.Backups(ctx, bIDs)
-	// TODO: print/log recoverable errors
-	if ferrs.Failure() != nil {
-		return Only(ctx, errors.Wrap(ferrs.Failure(), "Unable to retrieve backup results from storage"))
+	bups, berrs := r.Backups(ctx, bIDs)
+	if berrs.Failure() != nil {
+		return Only(ctx, errors.Wrap(berrs.Failure(), "Unable to retrieve backup results from storage"))
 	}
 
+	Info(ctx, "All Backups Complete:")
```

Comment:
nit: We will show "All Backups Complete:" even when some didn't. I feel like the following would be be a better way to phrase it.

```suggestion
	Info(ctx, "Completed backups:")
```